tensorflow_encrypted/tensor/native.py

```diff
@@ -90,6 +95,28 @@ def strided_slice(self, args: Any, kwargs: Any) -> 'NativeTensor':
     def reshape(self, axes: Union[tf.Tensor, List[int]]) -> 'NativeTensor':
         return NativeTensor(tf.reshape(self.value, axes), self.modulus)
 
+    def binarize(self) -> 'NativeTensor':
+        """Computes bit decomposition of tensor
+         tensor: ndarray of shape (x0, ..., xn)
+        returns: a binary tensor of shape (x0, ..., xn, bits) equivalent to tensor
+        """
+        if isinstance(self.value, np.ndarray):
```

Comment:
Is there a particular use case we have in mind for keeping numpy stuff in here? I feel like the `value` constructor arg could be converted to a tf tensor during construction if it's a numpy array

Reply:
Yeah was thinking about this a bit and it shouldn't really matter if its a numpy array or a tf tensor because the tf ops take numpy arrays as input.